store/db_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"database/sql"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+	"sync"
+	"testing"
+)
+
+func insertValue(db *DB, wg *sync.WaitGroup, t *testing.T) {
+	if err := db.Do(func(tx *sql.Tx) error {
+		// Get the current count.
+		rows, err := tx.Query("SELECT count(*) FROM rkt_db_test")
+		if err != nil {
+			return err
+		}
+		defer rows.Close()
+
+		var count int
+		for rows.Next() {
+			rows.Scan(&count)
+		}
+
+		// Increase the count.
+		_, err = tx.Exec(fmt.Sprintf("INSERT INTO rkt_db_test VALUES (%d)", count+1))
+		return err
+	}); err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+	wg.Done()
+}
+
+func getMaxValue(db *DB, t *testing.T) int {
+	var maxValue int
+	if err := db.Do(func(tx *sql.Tx) error {
+		// Get the current count.
+		rows, err := tx.Query("SELECT max(counts) FROM rkt_db_test")
+		if err != nil {
+			return err
+		}
+		defer rows.Close()
+
+		for rows.Next() {
```

Comment:
Or even better - split it out to separate function, as it could be used in `insertValue` too.